streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -167,6 +272,13 @@ public KStreamBuilder() {
     }
 
     /**
+     * Enum used to define auto offset reset for KStream instances.
```

Comment:
Also I felt it may better be in `TopologyBuilder` as it is used in both classes and `KStreamBuilder` extends from the former.